src/main/resources/synchronization/pl.yml

```diff
@@ -0,0 +1,50 @@
+# Minecraft -> Synchronizacja pseudonimów na Discordzie
+#
+# NicknameSynchronizationEnabled: czy automatycznie ustawić pseudonim użytkownika discorda na format pseudonimu
+# NicknameSynchronizationCycleTime: liczba minut między wielokrotnym uruchamianiem synchronizacji dla wszystkich graczy online
+# NicknameSynchronizationFormat: format pseudonimu (pamiętaj, że nie powinien przekraczać 32 znaków)
+# %displayname%: nazwa wyświetlana gracza
+# przykład: Jeb
+# %username%: nazwa użytkownika gracza
+# przykład: Jeb_
```

Comment:
```suggestion
# %displayname%:            nazwa wyświetlana gracza
#                             example: Jeb
# %username%:               nazwa użytkownika gracza
#                             example: Jeb_
# %discord_name%:           player's discord username
#                             example: Jeb
# %discord_discriminator%:  player's discord discriminator
#                             example: 4988
```
descriptions for `%discord_name%` and `%discord_discriminator%` need to be translated